.github/workflows/build-test.yml

```diff
@@ -569,7 +569,25 @@ jobs:
         ports:
           - 1025:1025
           - 8025:8025
+      manticoresearch:
+        image: manticoresearch/manticore:9.2.14
+        ports:
+          - 9308:9308
     steps:
+      # https://github.com/elastic/elastic-github-actions/blob/master/elasticsearch/README.md
+      - name: Configure sysctl limits for Elasticsearch
+        run: |
+          sudo swapoff -a
+          sudo sysctl -w vm.swappiness=1
+          sudo sysctl -w fs.file-max=262144
+          sudo sysctl -w vm.max_map_count=262144
+
+      - name: Runs Elasticsearch
+        uses: elastic/elastic-github-actions/elasticsearch@master
```

Comment:
not used?

Reply:
There is an elasticsearch unit test at packages/backend/server/src/plugins/indexer/__tests__/providers/elasticsearch.spec.ts being used, and we still need to continuously ensure that the Elasticsearch DSL is written correctly.